sdk/servicebus/service-bus/test/internal/batchReceiver.spec.ts

```diff
@@ -49,1102 +65,1073 @@ function afterEachTest(): Promise<void> {
   return serviceBusClient.test.afterEach();
 }
 
-describe("Batching Receiver", () => {
```

Comment:
Removed this outer describe block as it was not adding any value. No common code was here or any common before, after, beforeEach, afterEach functions either

Reply:
I prefer having the outer describe block since it would easily let me run all the tests in this file with a single `.only` on the describe block instead of having to put `.only` for each and every describe block of this file.

Reply:
Sounds good. Mind adding it back in your PR https://github.com/Azure/azure-sdk-for-js/pull/13929 since you will have to make changes to fix the merge conflict anyway?